Formula/x86_64-elf-gdb.rb

```diff
@@ -49,6 +53,9 @@ def install
     end
 
     mv include/"gdb", include/"x86_64-elf-gdb"
```

Comment:
I wonder if, in the spirit of what we're doing elsewhere, this should be
```suggestion
    (include/target).install include/"gdb"
```
or, more likely, we can drop this line entirely and just use `--includedir=#{include/target}` as a `configure` flag.

Reply:
Good point. `--includedir=#{include/target}` tests OK locally, so I'll push that after https://github.com/Homebrew/brew/pull/11597 is merged.